### PR TITLE
Remove newline at the end of .md files

### PR DESCRIPTION
**Pull request by huyvohcmc**
Remove newline at the end of markdown files #4370

### CODE REVIEW

**Comment by duailibe** [comment]
This is not the right fix.. we do want a new line at the end of the files, we just don't want them for empty files :)

File: tests/markdown_empty/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`empty.md 1`] = `
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`;
```
Comment (duailibe) [inline tests/markdown_empty/__snapshots__/jsfmt.spec.js.snap]: Every test that changed are regressions except for this one

File: tests/markdown_empty/not_empty.md [hunk 2]
```diff
@@ -0,0 +1 @@
+** Not empty **
```
Comment (duailibe) [inline tests/markdown_empty/not_empty.md]: This one isn't necessary. All of the other .md files are already examples of `not_empty.md`

You could also move `empty.md` to `tests/markdown/` so you don't have a single file in the folder, but it's fine either way.

File: tests/markdown_empty/__snapshots__/jsfmt.spec.js.snap (hunk 1 above)
Comment (duailibe) [inline tests/markdown_empty/__snapshots__/jsfmt.spec.js.snap]: Every test that changed are regressions except for this one

File: tests/markdown_empty/not_empty.md (hunk 2 above)
Comment (duailibe) [inline tests/markdown_empty/not_empty.md]: This one isn't necessary. All of the other .md files are already examples of `not_empty.md`

You could also move `empty.md` to `tests/markdown/` so you don't have a single file in the folder, but it's fine either way.

File: tests/markdown_empty/__snapshots__/jsfmt.spec.js.snap (hunk 1 above)
Comment (huyvohcmc) [reply tests/markdown_empty/__snapshots__/jsfmt.spec.js.snap]: I'll fix it later, could you add a WIP label or something?

Comment (duailibe) [reply tests/markdown_empty/__snapshots__/jsfmt.spec.js.snap]: Sure! Take your time

**Pull request closed without merging**